aspnetcore/grpc/services.md

```diff
@@ -328,6 +328,46 @@ A gRPC call ends on the server once the gRPC method exits. The following argumen
 
 If a gRPC method starts background tasks that use these types, it must complete the tasks before the gRPC method exits. Continuing to use the context, stream reader, or stream writer after the gRPC method exists causes errors and unpredictable behavior.
 
+In the following example, the server streaming method could write to the response stream after the call has finished:
```

Comment:
```suggestion
In the following example, the server streaming method writes to the response stream after the call has finished:
```

Reply:
I'll leave it as "could write" because it's not certain what will happen.